pkgs/development/python-modules/mkdocs-exclude/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, callPackage
+, buildPythonPackage
+, fetchFromGitHub
+, mkdocs
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-exclude";
+  version = "1.0.2";
+
+  # Repository has only 3 commits and no tags. Each of these commits has
+  # version of 1.0.0, 1.0.1 and 1.0.2 in setup.py, though.
```

Comment:
Maybe fetching from pypi would be better here.

Reply:
I trust git more. Reviewing pypi releases is more work for no win, given that `fetchFromGitHub` is downloading tarball too.

Reply:
Same for me but if upstream does not tag the releases correct. Whatever you prefer.